apps/users/views/auth.py

```diff
@@ -16,6 +27,10 @@ class LoginView(auth_views.LoginView):
     form_class = LoginForm
     template_name = 'users/login.html'
 
+    def get(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
+        """Respond to GET requests for the login page."""
+        return HttpResponsePermanentRedirect('/api/auth/signin')
```

Comment:
I don't see any urls using this view in [`/modularhistory/urls.py`](https://github.com/ModularHistory/modularhistory/blob/MH-11/modularhistory/urls.py), [`/apps/users/urls.py`](https://github.com/ModularHistory/modularhistory/blob/MH-11/apps/users/urls.py), or [`/apps/users/api/urls.py`](https://github.com/ModularHistory/modularhistory/blob/MH-11/apps/users/api/urls.py). Do we foresee this view being used anywhere?

Reply:
Oh, we can probably get rid of this. I don't foresee it being used, and it would be easy to recreate if necessary.